node/src/chain_spec/turing.rs

```diff
@@ -16,10 +16,9 @@ use turing_runtime::{
 
 static TOKEN_SYMBOL: &str = "TUR";
 const SS_58_FORMAT: u32 = 51;
-const TOTAL_TOKENS: u128 = DOLLAR * 1_000_000_000;
 static RELAY_CHAIN: &str = "rococo-local";
 static TURING_RELAY_CHAIN: &str = "ksmcc3";
-const DEFAULT_PARA_ID: u32 = 2000;
+const DEFAULT_PARA_ID: u32 = 2114;
```

Comment:
the default should be 2000 right? tghen that is the live one

Reply:
yep